hapi-postgres-connection error

Hello. I'm new to web development in general, and I started with nodejs. I'm using hapijs to generate the server. I'm also using postgresql 4 to design my database. 

Finally, I use hapi-postgres-connection plugin to connect to the database.

The first few times I tried it, it worked. 

I decided to use it with angularjs that I just started learning. Then it refused to work. This the code.

**const hapi_module = require('hapi');
const server = new hapi_module.Server();

server.connection({
	port:8080,
	host:'localhost',
	routes: {cors:true}
});

server.register ({
	register: require('hapi-postgres-connection')
	},
	function (error) {
		if (error) {
			throw error;
		}
}); 




server.route({
	path:'/postgresCountry',
	method: 'GET',
	handler: function(request, reply) {
		let email = 'postgres@postgres.net';
		let select = 'SELECT * FROM country';
		request.pg.client.query(select, function(error, result) {
			console.log(error, result.rows);
			console.dir(error);
			return reply(result.rows);
		});
	}
});**

The above is the hapijs code that is supposed to allow me access to the database. For some reason, it returns the following error:

**Server running at: http://localhost:8080

assert.js:81
  throw new assert.AssertionError({
  ^
AssertionError: hapi-postgres-connectionERROR Connecting to PostgreSQL!
    at C:\webstorm_nodejs\node_modules\hapi-postgres-connection\index.js:15:3
    at client.connect (C:\webstorm_nodejs\node_modules\pg-pool\index.js:219:9)
    at Connection.connectingErrorHandler (C:\webstorm_nodejs\node_modules\pg\lib\client.js:123:14)
    at emitOne (events.js:96:13)
    at Connection.emit (events.js:191:7)
    at Socket.<anonymous> (C:\webstorm_nodejs\node_modules\pg\lib\connection.js:71:10)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:191:7)
    at emitErrorNT (net.js:1279:8)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)**

I really don't know why this  error is thrown. I would really appreciate any help!

--- a/README.md
+++ b/README.md
@@ -101,6 +101,12 @@ or why this is a *good idea*
 (*hard-coding values in your app is a really bad idea...*)  
 please see: https://github.com/dwyl/learn-environment-variables
 
+### *Optional* Environment Variable: `DATABASE_SSL`
+If your database connection requires the use of SSL, you can set `DATABASE_SSL` environment
+variable to true and the pool connection will be done accordingly. This is required
+(for example) by databases hosted on [*Heroku*]
+(https://devcenter.heroku.com/articles/heroku-postgresql#heroku-postgres-ssl).
+
 ## *Q*: Don't We need to Close the Postgres Connection?
 
 ***A***: No! The plugin handles closing the connection for you!

--- a/index.js
+++ b/index.js
@@ -8,7 +8,7 @@ var PG_CON = []; // this "global" is local to the plugin.
 var run_once = false;
 
 // create a pool
-var pool = new pg.Pool({connectionString: process.env.DATABASE_URL});
+var pool = new pg.Pool({connectionString: process.env.DATABASE_URL, ssl: process.env.DATABASE_SSL || false});
 
 // connection using created pool
 pool.connect(function(err, client, done) {

